test/test_helper.exs

```diff
@@ -4,8 +4,7 @@ ExUnit.configure exclude: [integration: true, consumer_group: true, server_0_p_9
 
 defmodule TestHelper do
   def generate_random_string(string_length \\ 20) do
-    :random.seed(:os.timestamp)
-    Enum.map(1..string_length, fn _ -> (:random.uniform * 25 + 65) |> round end) |> to_string
+    Enum.map(1..string_length, fn _ -> (:rand.uniform * 25 + 65) |> round end) |> to_string
```

Comment:
@joshuawscott I don't know if we should be supporting it or not, you can make the call if you think we shouldn't support it please delete it here https://github.com/kafkaex/kafka_ex/blob/master/.travis.yml#L18-L19 and I'll merge